docs/transactions.md

```diff
@@ -834,10 +834,10 @@ Gets all child certificates for the given certificate.
 Revoked certificates are not returned.
 
 - Parameters:
-  - subject: `string`  - certificates's `Subject`
+  - subject: `string(Base64)`  - certificates's `Subject`
```

Comment:
Better option would be:
- subject: `string`  - certificates's `Subject` in base64 string format